pkgs/misc/urbit/libaes_siv/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, sources, cmake, openssl, enableParallelBuilding ? true }:
+
+stdenv.mkDerivation {
+  name = "libaes_siv";
+  version = sources.libaes_siv.rev;
+  src = sources.libaes_siv;
```

Comment:
This should ideally be moved into a package set wrapper

Reply:
I'm not sure I understand this